### ISSUE TITLE
Resume play from last position of previous medias

### ISSUE THREAD

**Issue by RachidTagzen** [nova-video-player/aos-AVP#208]
Hello,
The great feature to resume playing videos from their last time position is missing in AVP.
I hope that you add this feature like most other media players.
Thank you.

**Comment by courville** [comment]
Hi, resume from last played position is working on my end both on local storage and network shared. Could you please detail your specific test case? Thanks

**Comment by RachidTagzen** [comment]
I don't know if this feature is default or must be activated somewhere.
All my videos are on an USB HDD, and I play them on a H96 TV Box.
For example, when I watch a video and I go to settings to setup something, after that when I finish the video plays from beginning not from the position when I paused.

**Comment by courville** [comment]
OK I understand the issue.
Resume points are saved on a xml file and I guess that there is no write permission to the external USB HDD on your TV box for some reasons.
What is the filesystem type on your HDD, is it supported for write operation on your H96 box?

**Comment by RachidTagzen** [comment]
The file system is NTFS.
I can read and write in my HDD without any problem on Windows or Linux.
But I have no idea how it works on Android. And If remember i gave permission to AVP to access to files when it asked me like this :

![image](https://user-images.githubusercontent.com/25085777/66958676-96db6c80-f060-11e9-80f7-01116899bde5.png)

**Comment by courville** [comment]
Accessing and writing are two different things on Android.
And write support on NTFS is not always granted in all Android implementation.
Could you please check for me that you can delete a file on the HDD inside Nova application via long press on the file then delete? 
It will indicate if you indeed can write via Nova.
Also you can try via a file manager that is not a system app to check.

**Comment by RachidTagzen** [comment]
Deleting files inside Nova ends with failure.
The same case with "not system" files managers. They tell me that files are deleted, but in fact nothing is delete.
I tried MX Player (I have 2 favorite apps: MX Player & Nova Video Player :blush:), and I succeeded to delete files. But before that, MX Player asked me like this:
<br>
![image](https://user-images.githubusercontent.com/25085777/67043829-dd919b00-f122-11e9-9c01-191c525b9575.png)
<br>
![image](https://user-images.githubusercontent.com/25085777/67043803-c94d9e00-f122-11e9-9b17-f1554a3caf00.png)
<br>
I think that MX Player stores their "Resume points" in a file on the internal storage not in the external storage, because it remembers positions without asking me anything.
<br>
**PS: My device works with Android 8.**
